extensions/integration-tests/package.json

```diff
@@ -49,12 +49,13 @@
     "postinstall": "node ./node_modules/vscode/bin/install"
   },
   "devDependencies": {
-    "@types/node": "7.0.43",
     "@types/chai": "3.4.34",
+    "@types/node": "7.0.43",
+    "adstest": "github:ranasaria/adstest.git#0.0.2",
```

Comment:
>"adstest": "github:ranasaria/adstest.git#0.0.2", [](start = 4, length = 48)

Are you planning on doing PRs for this code?

Reply:
The code is already checked in, so not sure how to do a PR for that, but the code checked in there was already reviewed in previous iterations of this PR and has not changed at all except for some minor packaging aspects. If you can take a quick look at that on git hub and share any feedback in email then that would awesome.

---
In reply to: [288664339](https://github.com/microsoft/azuredatastudio/pull/5499#discussion_r288664339) [](ancestors = 288664339)